chess2.cpp

```diff
@@ -0,0 +1,882 @@
+#include "chess2.h"
+#include<iostream>
+
+int Chess2::perform(str Move) {
```

Comment:
Add comments. The function is not making it clear what is expected from the code and the expected input for it.
Also, whenever we make library like structure, its very important to have proper comments.